deepecho/models/par.py

```diff
@@ -210,17 +210,23 @@ def _data_to_tensor(self, data):
 
                 elif props['type'] in ['continuous', 'timestamp']:
                     mu_idx, sigma_idx, missing_idx = props['indices']
-                    x[mu_idx] = 0.0 if (data[key][i] is None or props['std'] == 0) else (
-                        data[key][i] - props['mu']) / props['std']
+                    if pd.isnull(data[key][i]) or props['std'] == 0:
```

Comment:
I like the change from one-liner to if / else block!
It improves readability a lot, in this case.